app/models/match.rb

```diff
@@ -81,4 +91,19 @@ def adjust_index_by(offset)
     self.index -= offset
     save!
   end
+
+  def calculate_points
+    return if !has_scores? || has_points?
+
+    if score_home > score_away
+      self.points_home = 3
+      self.points_away = 0
+    elsif score_home < score_away
```

Comment:
Shouldn't the win, lose and draw points be retrieved from the corresponding event? Or is this only possible after #197 is merged?

Reply:
yes and yes. But I can leave the hardcoded for now and you insert it ;)